### DISCUSSION TITLE
Running Monika on a server

### DISCUSSION THREAD

**Discussion by diondkb** [hyperjumptech/monika#1084] [General]
HI,

Can Monika be run on a server? Seems more reliable than running on a personal pc?

Thanks

**Comment by dennypradipta** [comment]
Sure you can. I've set up Monika on my servers too to monitor my web apps.

**Comment by diondkb** [comment]
Do you have a tutorial?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by diondkb** [comment]
Thanks for getting back

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
